src/main/java/org/launchcode/techjobs/oo/test/JobTest.java

```diff
@@ -1,12 +1,155 @@
 package org.launchcode.techjobs.oo.test;
 
+import org.junit.Assert;
+import org.junit.Test;
+//import org.junit.Before;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import org.launchcode.techjobs.oo.*;
+
+import static org.junit.Assert.*;
 
 /**
  * Created by LaunchCode
  */
 @RunWith(JUnit4.class)
+
 public class JobTest {
+    Job job1;
```

Comment:
I don't believe you need these variables here. You've declared new variables in each of your methods, making them local to the methods. The global variables here are no longer uses. (see comment below). Doesn't hurt your code, just not necessary.